src/sinol_make/commands/inwer/__init__.py

```diff
@@ -49,15 +50,17 @@ def verify_test(self, execution: InwerExecution) -> VerificationResult:
         """
         output_dir = os.path.join(os.getcwd(), 'cache', 'executions', execution.test_name)
         os.makedirs(output_dir, exist_ok=True)
-        output_file = os.path.join(output_dir, package_util.get_file_name_without_extension(execution.inwer_exe_path) + ".out")
 
-        command = f'{execution.inwer_exe_path} < {execution.test_path} > {output_file}'
-        exit_code = os.system(command)
+        command = f'{execution.inwer_exe_path} < {execution.test_path}'
+        process = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+        process.wait()
+        exit_code = process.returncode
+        out, _ = process.communicate()
```

Comment:
Does `out` also capture `stderr`? The `assert` output should be included in the output, so that the user would know which assert etc. has been triggered.

Reply:
Yes, there is `stderr=subprocess.STDOUT`